providers/statsig/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/open-feature/go-sdk-contrib/providers/statsig
+
+go 1.19
```

Comment:
You could consider upgrading to 1.21 . Go 1.22 is already out and soon we plan to upgrade go-sdk to 1.20 (and 1.21 eventually)

Reply:
done